docs/tasks/stateful-sets/deleting-a-statefulset.md

```diff
@@ -0,0 +1,102 @@
+---
+assignees:
+- bprashanth
+- erictune
+- foxish
+- janetkuo
+- smarterclayton
+
+---
+
+{% capture overview %}
+This task shows you how to delete a Stateful Set.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* If you are not using Stateful Sets, you can skip this task now. 
+
+{% endcapture %}
+
+{% capture steps %}
+
+### Deleting a StatefulSet
+
+A Stateful Set can be deleted like other resources in kubernetes. 
+
+```shell
```

Comment:
Wow, thanks for pointing this out. I need to update my doc to match this guide also.